site/app/libraries/database/DatabaseQueries.php

```diff
@@ -6241,6 +6241,11 @@ public function getQueueId($queue_code) {
         return $this->course_db->rows()[0]['id'];
     }
 
+    public function getQueueHasContactInformation($queue_code) {
+        $this->course_db->query("select * from queue_settings where code = ?;", [$queue_code]);
```

Comment:
Is this query guaranteed to always return something, should there be a check for the number of results returned or ability to handle a null response

Reply:
The column has a default value of true and there is no way to ever make it something other than true/false so it's guaranteed to return something but I guess a check can't hurt

Reply:
Ah ok if there's a default value then I expect it to be safe, wasn't sure if this was a value that could ever be unset